src/main/java/com/oneandone/snmpman/SnmpmanAgent.java

```diff
@@ -299,6 +272,40 @@ protected void registerManagedObjects() {
         }
     }
 
+    /** Reads a walk file line by line parsing the contents.
+     * @param walkFile the file name containing lines in the {@link #VARIABLE_BINDING_PATTERN} format.
+     * @return the parsed variable bindings from the file.
+     */
+    private Map<OID, Variable> readWalkFile(final File walkFile) throws IOException {
+        final Map<OID, Variable> bindings = new HashMap<>();
+        
+        List<String> lines = Files.readAllLines(walkFile.toPath(), Charset.forName("UTF-8"));
```

Comment:
File may be large and will be read completly into memory and processed afterwards, we prefer the old way